lib/rails_rinku.rb

```diff
@@ -13,10 +13,18 @@ def rinku_auto_link(text, *args, &block)
     options.reverse_merge!(:link => :all, :html => {})
     text = h(text) unless text.html_safe?
 
+    tag_options_method = if Rails.version.starts_with?('5') && Rails.version.split('.')[1].to_i > 0
```

Comment:
You can just check with `Rails.version >= '5.1'`, or similar. This will also prevent the gem breaking with Rails 6 and above.